pipeline_steps/common.groovy

```diff
@@ -1915,6 +1919,44 @@ void createRelease(String component_text, Boolean from_rc_branch){
   )
 }
 
+void updateDependents(REPO_NAME){
```

Comment:
I'm not a fan of `REPO_NAME` either.  I kept it REPO_NAME so it's easy to know where it came from, maybe the original use of REPO_NAME should also be renamed [here](https://github.com/jlpogue3/rpc-gating/blob/c8964ac63e06710f2410d3db85840ef558338bf5/pipeline_steps/common.groovy#L1574-L1575)?
If it should be renamed, I'm hesitant to rename it "componentName" just so it can be differentiated as the component being released, perhaps something like "releaseComponentName"